### ISSUE TITLE
Any documentation or examples

### ISSUE THREAD

**Issue by gta191977649** [TheNormalnij/Streamer#1]
Hi, is there any documentation or example of how to use this streamer?

**Comment by TheNormalnij** [comment]
Yeah, im going to add examples and documentation later, when the tool will be stable.
Now it works in debug mode:
1. Create a world resource with using [the converter ](https://github.com/TheNormalnij/MtaWorldConvertingTools)
2. Start streamer and the world resource
3. Load world `/loadworld <world name>`
